src/main/java/codeenthusiast/TrainingCenterApp/exercise/strengthexercise/keytechniqueelement/KeyTechniqueController.java

```diff
@@ -0,0 +1,20 @@
+package codeenthusiast.TrainingCenterApp.exercise.strengthexercise.keytechniqueelement;
+
+import codeenthusiast.TrainingCenterApp.abstracts.AbstractController;
+import codeenthusiast.TrainingCenterApp.abstracts.AbstractService;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class KeyTechniqueController extends AbstractController<KeyTechniqueElement, KeyTechniqueElementDTO> {
```

Comment:
You should stick to one version of KeyTechnique : KeyTechnique or KeyTechniqueElements